monika.example.yml

```diff
@@ -33,12 +33,8 @@ probes:
 #   name: ping_test
 #   description: requesting icmp ping
 #   interval: 10
-#   requests:
+#   ping:
 #     - url: http://google.com
```

Comment:
It is stated [in the documentation](https://monika.hyperjump.tech/guides/probes#ping-request) that it uses uri instead or url.

Reply:
Good catch !

Fixed